### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by nissaba** [twelve17/openalpr-ios#7]
afther installing the pods. Some files where missing the import for the version.h file. 
also some files are missing from the project such as :

- featurematcher.cpp (bad project reference) 
- featurematcher.h (bad project reference) 
- postprocess.h and postprocess.h (bad project reference) 
- stateidentifier.h and  stateidentifier.cpp (Missing)
- trex.h and trex.c (missing)
- runtime_data (missing)

**Comment by mobileosdeveloper** [comment]
Myself also having same issue .... Any solution to fix this issue?

**Comment by nissaba** [comment]
Needed to pull one of the more advanced forks.

**Comment by twelve17** [comment]
For v2 of this project, I have replaced the old instructions and bash scripts with a Ruby script to build the OpenALPR and dependency Frameworks, as well as the OpenALPR Xcode project.  A sample app is included.  These changes have been pushed to the master branch, and the old version has been moved to branch `1_0`.  Feel free to check out this new script and see if it works for you.  I will no longer be working on or supporting the `1_0` codebase.
